docs/docs/zh/guide/datasource/teradata.md

```diff
@@ -0,0 +1,15 @@
+# TERADATA 数据源
```

Comment:
Did you translate the Chinese doc from translator like google translate? :laughing

Reply:
No, just copy/paste... :) Is this wrong?

Reply:
no, that great 👍